tests/tests/test_cluster.py

```diff
@@ -278,3 +279,59 @@ class Meta:
         self.assertEqual(error.id, 'fields.E300')
         self.assertEqual(error.obj, Instrument.banana.field)
         self.assertEqual(error.msg, "Field defines a relation with model 'Banana', which is either not installed, or is abstract.")
+
+    def test_parentalm2mfield(self):
+        article = Article(title="Test Title")
+        author_1 = Author(name="Author 1")
+        author_2 = Author(name="Author 2")
+        article.authors = [author_1, author_2]
+        category_1 = Category(name="Category 1")
+        category_2 = Category(name="Category 2")
+        article.categories = [category_1, category_2]
+        self.assertEqual(['Author 1', 'Author 2'],
+                         [author.name for author in article.authors.all()])
```

Comment:
We can't guarantee the ordering here (well, we probably _can_ guarantee it for our fake querysets, but not live ones coming from the database) - there should probably be an `order_by('name')` here, or else we should compare them as sets.